src/clr_bundle_ls.c

```diff
@@ -133,7 +138,20 @@ static bool parse_options(int argc, char **argv)
 			}
 			set_cert_path(optarg);
 			break;
-
+		case 'D':
+			if (!optarg) {
+				fprintf(stderr, "Invalid --has-dep argument\n\n");
+				goto err;
+			}
+			string_or_die(&cmdline_option_has_dep, optarg);
```

Comment:
For safety, you should do `string_or_die(&cmdline_option_has_dep, "%s", optarg)`, since `optarg` is a variable set from user input.

Reply:
Fixed.